Gemfile

```diff
@@ -73,3 +74,5 @@ end
 gem 'tzinfo-data', platforms: [:mingw, :mswin, :x64_mingw, :jruby]
 
 gem 'wcmc_components', path: 'wcmc_components'
+
+gem 'wcmc_devise_sso', path: 'wcmc_devise_sso'
```

Comment:
Just for my understanding, why do we have `line 8` and `line 78` (same gem with different paths) ??

Reply:
Thanks! Have deleted these then 👍